src/Component/EachItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+
+
+function EachItem ({ item, addCart, subtotal }){
+    const { name, price, img, description} = item;
+    return(
+        <div className="EachItem">
+            <h2><strong>{name}</strong></h2>
+            <p>Price: ${price}</p>
```

Comment:
The price on this line must have to digits after the decimal when it's rendered in the browser. This is another spot where `formatPrice` might come in handy!